Build error in CI environment after updating sentry-cocoa

```
Run git submodule update --init --recursive modules/sentry-cocoa
[6](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:7)
Submodule 'modules/sentry-cocoa' (https://github.com/getsentry/sentry-cocoa) registered for path 'modules/sentry-cocoa'
[7](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:8)
Cloning into '/Users/runner/work/sentry-unreal/sentry-unreal/modules/sentry-cocoa'...
[8](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:9)
Submodule path 'modules/sentry-cocoa': checked out '3e56a8dd14f4adc4b7de7ce6517f91543bf2046c'
[9](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:10)
*** xcodebuild output can be found in /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/carthage-xcodebuild.UmhDOt.log
[10](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:11)
*** Building scheme "Sentry" in Sentry.xcworkspace
[11](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:12)
cp: modules/sentry-cocoa/Carthage/Build/Sentry.xcframework/ios-arm64_armv7/Sentry.framework: No such file or directory
[12](https://github.com/getsentry/sentry-unreal/actions/runs/3391866170/jobs/5637437257#step:7:13)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/sdk.yml
+++ b/.github/workflows/sdk.yml
@@ -16,7 +16,7 @@ jobs:
       run:
         shell: bash
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Select submodule
         id: env
@@ -35,7 +35,7 @@ jobs:
       - name: Get submodule status
         run: git submodule status --cached ${{ steps.env.outputs.submodule }} | tee submodule-status
 
-      - uses: actions/cache@v2
+      - uses: actions/cache@v3
         id: cache
         with:
           path: |

--- a/scripts/build-ios.sh
+++ b/scripts/build-ios.sh
@@ -8,7 +8,7 @@ rm -rf "${sentryArtifactsDestination}/"*
 
 carthage build --project-directory "${sentryIosRoot}" --use-xcframeworks --no-skip-current --platform iOS
 
-cp -R "${sentryIosRoot}/Carthage/Build/Sentry.xcframework/ios-arm64_armv7/Sentry.framework" "${sentryArtifactsDestination}/Sentry.framework"
+cp -R "${sentryIosRoot}/Carthage/Build/Sentry.xcframework/ios-arm64/Sentry.framework" "${sentryArtifactsDestination}/Sentry.framework"
 
 mkdir "Sentry.embeddedframework"
 

